src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/Sql/OdbcCommandWrapper.cs

```diff
@@ -2,45 +2,48 @@
 // SPDX-License-Identifier: Apache-2.0
 
 using System;
-using System.Data.Odbc;
 using NewRelic.Agent.Api;
 using NewRelic.Agent.Extensions.Providers.Wrapper;
 using NewRelic.Agent.Extensions.Parsing;
 using NewRelic.Agent.Extensions.Parsing.ConnectionString;
+using System.Data;
 
 namespace NewRelic.Providers.Wrapper.Sql
 {
-	public class OdbcCommandWrapper : IWrapper
-	{
-		public const string WrapperName = "OdbcCommandTracer";
-		public bool IsTransactionRequired => true;
+    public class OdbcCommandWrapper : IWrapper
+    {
+        public const string WrapperName = "OdbcCommandTracer";
+        public bool IsTransactionRequired => true;
 
-		public CanWrapResponse CanWrap(InstrumentedMethodInfo methodInfo)
-		{
-			return new CanWrapResponse(methodInfo.RequestedWrapperName.Equals(WrapperName, StringComparison.OrdinalIgnoreCase));
-		}
+        public CanWrapResponse CanWrap(InstrumentedMethodInfo methodInfo)
+        {
+            return new CanWrapResponse(methodInfo.RequestedWrapperName.Equals(WrapperName, StringComparison.OrdinalIgnoreCase));
+        }
 
-		public AfterWrappedMethodDelegate BeforeWrappedMethod(InstrumentedMethodCall instrumentedMethodCall, IAgent agent, ITransaction transaction)
-		{
-			{
-				var odbcCommand = (OdbcCommand)instrumentedMethodCall.MethodCall.InvocationTarget;
-				if (odbcCommand == null)
-					return Delegates.NoOp;
+        public AfterWrappedMethodDelegate BeforeWrappedMethod(InstrumentedMethodCall instrumentedMethodCall, IAgent agent, ITransaction transaction)
+        {
+            {
+                var odbcCommand = (IDbCommand)instrumentedMethodCall.MethodCall.InvocationTarget;
```

Comment:
A hard cast like this will explode with an exception if the type can't be cast to the destination. Apparently that's never happened, though, as I see the old code did a hard cast to `OdbcCommand` and a null check the same way. Maybe we got lucky. 

In any case, I'd suggest a safer cast with some new-age syntactic dressing: 
```
                if (instrumentedMethodCall.MethodCall.InvocationTarget is not IDbCommand odbcCommand)
                    return Delegates.NoOp;
```